javascript-bank/account.js

```diff
@@ -0,0 +1,44 @@
+/* exported Account */
+
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  if (amount > 0 && Number.isInteger(amount)) {
+    var aTransaction = new Transaction();
+    aTransaction.type = 'deposit';
+    aTransaction.amount = amount;
```

Comment:
Make use of the `Transaction` constructor's parameters, instead of manually tweaking its properties.